app/domain/authentication/authn_jwt/input_validation/validate_claim_name.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Validate the claim name value
+      ValidateClaimName ||= CommandClass.new(
+        dependencies: {
+          regexp_class: Regexp,
+          deny_claims_list_value: [],
```

Comment:
will this be set as input in the app code or just in UTs?

Reply:
In app code